server/src/main/java/com/flipkart/varadhi/VaradhiApplication.java

```diff
@@ -48,31 +47,33 @@ public static void main(String[] args) {
             String memberId = configuration.getMember().getMemberId();
             CoreServices services = new CoreServices(configuration);
             VaradhiZkClusterManager clusterManager = getClusterManager(configuration, memberId);
-            Map<ComponentKind, Component> components = getComponents(configuration, services, clusterManager);
-
-            createClusteredVertx(configuration, clusterManager, services, "127.0.0.1").compose(vertx ->
-                    Future.all(components.entrySet().stream()
-                            .map(es -> es.getValue().start(vertx).onComplete(ar -> {
-                                if (ar.succeeded()) {
-                                    log.info("component: {} started.", es.getKey());
-                                } else {
-                                    log.error("component: {} failed to start. {}", es.getKey(), ar.cause());
-                                }
-                            })).collect(Collectors.toList()))
-            ).onComplete(ar -> {
-                if (ar.succeeded()) {
-                    log.info("VaradhiApplication Started on {}.", host);
-                } else {
-                    log.error("VaradhiApplication on host {} failed to start. {} ", host, ar.cause());
-                }
-            });
+            Map<ComponentKind, Verticle> verticles = getComponentVerticles(configuration, services, clusterManager);
+            createClusteredVertx(configuration, clusterManager, services, host).compose(vertx ->
```

Comment:
host here would be of the format <hostname>.local on mac laptop. By default hostname.local names are not getting resolved from the code. Workaround is to add the same to /etc/hosts.